PurrSoft/PurrSoft.Api/Controllers/AnimalController.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using PurrSoft.Api.Controllers.Base;
+using PurrSoft.Application.Commands.AnimalCommands;
+using PurrSoft.Application.Common;
+using PurrSoft.Domain.Entities;
+using System.Net;
+
+namespace PurrSoft.Api.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public class AnimalController : BaseController
+{
+    public AnimalController()
+    {
+    }
+
+    [HttpGet]
+    [Authorize(AuthenticationSchemes = "Bearer")]
```

Comment:
Think if you really need this to be with authentication . I would assume you can see the animals without it. So maybe make it anonymous?